src/OpenTelemetry/Metrics/MetricStreamConfiguration.cs

```diff
@@ -14,8 +14,16 @@
 // limitations under the License.
 // </copyright>
 
+using System;
+using System.Collections.Generic;
+
 namespace OpenTelemetry.Metrics
 {
+    // TODO: Clarify if spec allows adding anything from context
+    // or only Baggage entries. If the later, we just need to accept the
+    // Baggage key names only, and SDK can internally do Baggage.GetBaggage(name)
+    public delegate ReadOnlySpan<KeyValuePair<string, object>> AdditionalTagsCallback();
```

Comment:
@reyang need some input from you here.

Reply:
I think it is `extraDimensionsFromBaggage` and `extraDimensionsFromContext`.

Reply:
I suggest we focus on baggage for now. The OpenTelemetry Context is using opaque key rather than string, which gives additional challenge.